internal/fvalidate/verapdf_validator.go

```diff
@@ -82,3 +83,32 @@ func (v *veraPDFValidator) Validate(path string) (string, error) {
 		return "", errors.New("PDF/A validation failed with an application error")
 	}
 }
+
+func RunCommand(name string, args ...string) (string, error) {
+	result := exec.Command(name, "--version") // #nosec: G204 -- trusted path.
+
+	output, err := result.Output()
+	if err != nil {
+		return "", err
+	}
+
+	return string(output), nil
+}
+
+var Run = RunCommand
```

Comment:
Ideally, this would be a dependency injected when the validator is created, to avoid replacing it in tests. No need to do this now, but it would be nice to create an interface that accepts an `exec.Command` or a mocked version, so you can pass that directly and not have a default function.